lib/log.js

```diff
@@ -94,17 +97,18 @@ Lug.prototype.summary = function (request, response) {
     agent: request.headers['user-agent'],
     remoteAddressChain: request.app.remoteAddressChain,
     accountRecreated: request.app.accountRecreated,
-    t: Date.now() - request.info.received
+    t: Date.now() - request.info.received,
+    uid: credentials.uid || payload.uid || query.uid || response.uid || responseBody.uid || '00',
+    service: payload.service || query.service,
+    reason: payload.reason || query.reason,
+    redirectTo: payload.redirectTo || query.redirectTo,
+    keys: query.keys,
+
+    // Additional data used by the DataFlow fraud detection pipeline
```

Comment:
Should we mention anything about PII here and why it is ok?

Reply:
Good idea.